Makefile

```diff
@@ -68,6 +69,16 @@ build-path:
 
 serve: | front-build back-start
 
+compose-serve: | require-repos-folder front-dependencies build
+	REPOS_FOLDER=${REPOS_FOLDER} \
```

Comment:
question: should this use the same GITBASEPG prefix as other env vars? It's not used by gitbase-playground itself, but it's used by the docker compose file we provide so... I'm not sure.